data/mods/ssb/abilities.js

```diff
@@ -98,6 +98,34 @@ let BattleAbilities = {
 			this.boost({atk: 1, def: -1, spd: -1, spe: 1});
 		},
 	},
+	// Akiamara
+	toxicswap: {
+		shortDesc: "On switch-in, this Pokemon swaps all stat changes with the opponent. Ignores abilities.",
+		isNonstandard: "Custom",
+		id: "toxicswap",
+		name: "Toxic Swap",
+		onStart(pokemon) {
+			let target = pokemon.side.foe.active[0];
+			if (!target) return;
+			let targetBoosts = {};
+			let pokemonBoosts = {};
+
+			// @ts-ignore
+			for (let i in target.boosts) {
+				// @ts-ignore
+				targetBoosts[i] = target.boosts[i];
+				// @ts-ignore
+				pokemonBoosts[i] = pokemon.boosts[i];
+			}
+			target.setBoost(pokemonBoosts);
+			pokemon.setBoost(targetBoosts);
+
+			this.add('-swapboost', pokemon, target, '[from] move: Heart Swap');
```

Comment:
Shouldn't be `[from] move: Heart Swap`, you can exclude it and use a custom this.add after this line for a custom message.